src/common/logging.cpp

```diff
@@ -150,7 +169,15 @@ void otbrLogv(otbrLogLevel aLevel, const char *aFormat, va_list aArgList)
 
 void otbrLogvNoFilter(otbrLogLevel aLevel, const char *aFormat, va_list aArgList)
 {
-    vsyslog(static_cast<int>(aLevel), aFormat, aArgList);
+    if (syslogDisabled)
+    {
+        vprintf(aFormat, aArgList);
+        printf("\n");
```

Comment:
I think in this way it implies printing logs to standard output when syslog is disabled. I'd suggest to document the behavior somewhere.

Also I wonder if logging to standard output should be controlled by an independent flag from whether logging to syslog? It's potentially useful to disable all logging outputs.

Reply:
At least I think we should document the behavior (when syslog is off, write to stdout) in the comments otherwise later we'll have to read the code to understand what it does.

Reply:
Tried to do that here: https://github.com/openthread/ot-br-posix/blob/1e0a48f75320ca8056557827bb3962eada4eb8f1/src/agent/main.cpp#L142

I can put a comment around this as well.